modules/simulations/k8s-managed/requirements-dev.txt

```diff
@@ -1,2 +1,4 @@
-apache-airflow~=2.2.4
-airflow-kubernetes-job-operator~=2.0.4
```

Comment:
You have replaced `k8s-job-operator` with an incorrect one. can you find the right version for k8s?

Reply:
I will put it back to where it was....